### ISSUE TITLE
Nil for some videos

### ISSUE THREAD

**Issue by umairgillaniucp** [devandsev/YoutubeDirectLinkExtractor#3]
https://www.youtube.com/watch?v=xb1bSqF3u98

returning nil for this video

**Comment by n2waf** [comment]
same issue
some video return nil

**Comment by HoangDinhTho** [comment]
same issue with video return nil
https://www.youtube.com/watch?v=ZG5uAns-_yU

**Comment by akashRaghani92** [comment]
Yes facing the same issue with many URLS return nil AND not able to extract video but same video is playing in webbrowser!!

**Comment by acotilla91** [comment]
Here's another example: https://www.youtube.com/watch?v=t8G5ADTB17s
